packages/communication-react/tsconfig.json

```diff
@@ -10,7 +10,8 @@
       "@internal/calling-stateful-client": ["../../calling-stateful-client/src"],
       "@internal/chat-stateful-client": ["../../chat-stateful-client/src"],
       "@internal/react-components": ["../../react-components/src"],
-      "@internal/react-composites": ["../../react-composites/src"]
+      "@internal/react-composites": ["../../react-composites/src"],
+      "@internal/northstar-wrapper": ["../../northstar-wrapper/src"]
```

Comment:
heads up we might have to add this at a future time to the storybook and sample app tsconfigs&webpack configs -- but lets only do that when/if we need to